lib/kernel/src/code_server.erl

```diff
@@ -1280,6 +1246,12 @@ with_cache(cache, Dir, ModFile) ->
 with_cache(Cache, _Dir, ModFile) when is_map(Cache) ->
     {is_map_key(ModFile, Cache), Cache}.
 
+absname_when_relative(File) ->
+    case filename:pathtype(File) of
+        absolute -> File;
```

Comment:
I think this makes the `absolute` clause in the `absname/2` function dead

Reply:
Yeah, I want to look at this later, this is just to avoid duplication. I am taking notes, if you have more suggestions, let me know. :)